agent/agent-profiler/agent-service-profiler/src/main/java/com/microsoft/applicationinsights/serviceprofilerapi/profiler/JfrProfiler.java

```diff
@@ -81,6 +81,8 @@
   private final RecordingConfiguration memoryRecordingConfiguration;
   private final RecordingConfiguration cpuRecordingConfiguration;
 
+  private final String tempDirectory = System.getProperty("java.io.tmpdir");
```

Comment:
something like this (you can change `profiles` name to something else you prefer) will be more similar to temporary locations of other files that we store (temporary local disk storage for telemetry if ingestion sporadically fails)
```suggestion
  private final File tempDirectory = new File(LocalFileSystemUtils.getTempDir(), "applicationinsights/profiles")
```